torch/_decomp/__init__.py

```diff
@@ -260,6 +260,7 @@ def core_aten_decompositions() -> Dict[torch._ops.OperatorBase, Callable]:
             aten.addcmul_,
             aten.addr,
             aten.affine_grid_generator,
+            aten.alias_copy.default,
```

Comment:
```suggestion
            aten.alias_copy,
```
We are covering all overloads (default and out). This should fix the errors.